modules/luci-mod-dashboard/htdocs/luci-static/resources/view/dashboard/css/custom.css

```diff
@@ -0,0 +1,353 @@
+
+/**
+ * Bootstraps Basics Styles
+**/
+
+.label {
+    display: inline;
+    padding:2px 3px 2px 3px;
+    font-size: 75%;
+    font-weight: 700;
+    line-height: 1;
+    color: #fff;
+    text-align: center;
+    white-space: nowrap;
+    vertical-align: baseline;
+    border-radius: .25em;
+    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+}
+
+.label-success {
+    background-color: #5cb85c;
+}
+
+.label-danger {
+    background-color: #d9534f;
+}
+
+.progress {
+    display: -ms-flexbox;
+    display: flex;
+    height: 1rem;
+    overflow: hidden;
+    font-size: .75rem;
+    background-color: #e9ecef;
+    border-radius: .25rem;
+}
+
+.progress-bar {
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    -ms-flex-pack: center;
+    justify-content: center;
+    color: #fff;
+    text-align: center;
+    white-space: nowrap;
+    background-color: #007bff;
+    transition: width .6s ease;
+}
+
+.bg-success {
+    background-color: #28a745!important;
+}
+
+.bg-warning {
+    background-color: #ffc107!important;
+}
+
+.bg-danger {
+    background-color: #dc3545!important;
+}
+
+/**
+ * Dashboard Principals Styles
+**/
+
+.Dashboard {
+    color: #212529!important;
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,"Noto Sans",sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji";
```

Comment:
This line is very long. Can this line be broken in any way?
Do we need this a font-families?

Reply:
It looks slightly different but still fine. I don't see a problem removing it.

Reply:
I just noticed that there are already a lot of fonts. These do not have to be.

Reply:
Yes, in theory, it is enough that we keep 2~3 common system fonts